### ISSUE TITLE
Code style

### ISSUE THREAD

**Issue by jpcima** [sfztools/sfizz#79]
It's not clear which code style must be adopted for contributing.
It would be nice to settle for one, and introduce a clang-format configuration for the whole source.

The most frequent that I can see is reminiscent of Stroustrup's style from C++ guidelines.

But there's some files, like `lv2/sfizz.c`, which mix different kinds of "*" and "{" placement, and make it extremely confusing.

**Comment by paulfd** [comment]
IIRC the C++ part should be close to clang-format "WebKit". I'm easily influenced for these things, so since I started this project with JUCE I tended to use their naming approach, and when coding the LV2 part I fell back to the more usual C style naming that I also use in Python.

I have no strong feeling on these matters, so I agree with the idea of settling for one. If it's a default and common clang-format one, even better.
